src/components/customErrorPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from "react"
+
+import Button from "./button"
+
+interface Props {
+  /**
+   * Any status code number
+   */
+  statusCode: number | "default"
+  /**
+   * the source path to the image you want to show
+   */
+  image: string
+  /**
+   * the page title as an h1 heading
+   */
+  pageTitle: string
+  /**
+   * the page subtitle as an h2 heading
+   */
+  pageSubtitle: string
+  /**
+   * the redirection path on click of the button
+   */
+  redirectHref: string
+  /**
+   * the label for the redirection button
+   */
+  redirectLabel: string
+}
+
+const CustomErrorPage: FC<Props> = ({
+  statusCode,
+  image,
```

Comment:
I would rather encapsulate the image. if we pass everything, there is little value in extracting in the first place

Reply:
I can also pass it as a function if that is better?